interface/main/tabs/menu/menus/standard.json

```diff
@@ -1230,22 +1230,39 @@
               "super"
             ],
             "global_req": [
-              "erx_enable",
-              "newcrop_user_role"
+              "erx_enable"
             ]
           },
           {
             "label":"Weno Management",
             "menu_id":"adm0",
             "target":"adm",
-            "url":"/interface/weno/admin.php",
+            "url":"/interface/weno/facilities.php",
             "children":[],
             "requirement":0,
             "acl_req": [
               "admin",
-              "super"
-            ]
-          }
+              "patients"
+            ],
+              "global_req": [
+                  "weno_rx_enable"
+              ]
+          },
+            {
+                "label":"Prescription Management",
+                "menu_id":"adm0",
+                "target":"adm",
+                "url":"/interface/weno/rxlogmanager.php",
+                "children":[],
+                "requirement":0,
+                "acl_req": [
+                    "patients",
+                    "rx"
```

Comment:
Since have decided to make this an admin function (per other comment regarding the rxlogmanager.php script, then this aco should be 'admin' => 'super'